pkg/runner/cluster_k8s.go

```diff
@@ -341,7 +342,7 @@ func (c *ClusterK8sRunner) Run(ctx context.Context, input *api.RunInput, ow *rpc
 						return err
 					}
 
-					fmt.Print(logs)
+					ow.WriteProgress([]byte(logs))
```

Comment:
We know this won't scale well for a large number of instances, but for the purposes of the Lotus tests it will be fine.